_health-care/_js/components/personal-information/AdditionalInformationSection.jsx

```diff
@@ -12,11 +14,10 @@ class AdditionalInformationSection extends React.Component {
 
         <div className="row">
           <div className="small-12 columns">
-            <input
-                type="checkbox"
-                id="veteran_is_essential_aca_coverage"
-                name="veteran_is_essential_aca_coverage"/>
-            <label htmlFor="veteran_is_essential_aca_coverage">I am enrolling to obtain minimal essential coverage under the affordable care act</label>
+            <ErrorableCheckbox
+                label="I am enrolling to obtain minimal essential coverage under the affordable care act"
+                value={this.props.data.isEssentialAcaCoverage}
+                onValueChange={(update) => {this.props.onStateChange('isEssentialAcaCoverage', update);}}/>
```

Comment:
Ah...this is why we were passing that string around. Good enough for this PR, but heads up, we actually need to go through and undo the pattern of declaring functions inside the attributes like we did here.

Apparently this causes JS to create a new function every time the component is rendered which is silly.